app/models/user.py

```diff
@@ -17,6 +19,55 @@ class User(baseModel):
 
     def __str__(self):
         return str(self.__dict__)
+    
+    def __init__(self,*args, **kwargs):
+        super().__init__(*args,**kwargs)
+        self._ldsCache = None  
+
+    @property
+    def isLaborDepartmentStudent(self):
+        if self._ldsCache is not None:  
+            return self._ldsCache
+            
+        if not self.student:
+            self._ldsCache = False
+            return self._ldsCache
+
+        from app.models.laborStatusForm import LaborStatusForm
+        from app.models.formHistory import FormHistory
+        today = date.today()
+
+        # Subquery to check if student has an approved release form
+        released_forms = (
+            FormHistory
+            .select(FormHistory.formID)
+            .where(
+                (FormHistory.releaseForm.is_null(False)) &  # Has a release form
+                (FormHistory.status == "Approved")     # Release form is approved
```

Comment:
Let's go ahead and add in a join to `LaborStatusForm` and check `studentSupervisee` as well, so our list is release forms for the student, and not release forms for every student ever.